app/controllers/posts_controller.rb

```diff
@@ -149,15 +155,17 @@ def update
 
     change_status and return if params[:status].present?
     change_authors_locked and return if params[:authors_locked].present?
+
+    @post.assign_attributes(post_params(params[:button_preview].blank?))
+    @post.written.assign_attributes(written_params)
     preview and return if params[:button_preview].present?
 
-    @post.assign_attributes(post_params)
     @post.board ||= Board.find(3)
     settings = process_tags(Setting, :post, :setting_ids)
     warnings = process_tags(ContentWarning, :post, :content_warning_ids)
     labels = process_tags(Label, :post, :label_ids)
 
-    if current_user.id != @post.user_id && @post.audit_comment.blank? && !@post.author_ids.include?(current_user.id)
+    if current_user.id != @post.user_id && @post.written.audit_comment.blank? && @post.audit_comment.blank? && !@post.author_ids.include?(current_user.id)
```

Comment:
Why do we have two of these...? Do we give two boxes for edit notes when you update a post?

Reply:
...I don't know. ...I might have just assumed we'd have one tbqh